orc8r/tools/packer/scripts/upgrade-ovs-patch-1.1.20210618.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+################################################################################
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+set -euo pipefail
+set -x
+
+export version="2.15.4-9-magma"
+export DEBIAN_FRONTEND="noninteractive"
+
+sudo rm -rf /etc/apt/sources.list.d/*.list
+sudo rm -rf /var/cache/apt/
+
+echo "Installing certificates ..."
+sudo apt-get update
+sudo apt-get install -y ca-certificates
+
+echo "Adding artifactory ..."
+echo 'deb https://artifactory.magmacore.org/artifactory/debian-test focal-ci main' | sudo tee /etc/apt/sources.list.d/magma.list
```

Comment:
I'd usually prefer to do it with system tools, as this adds some assertions and minimizes the possibility to do something wrong.
Here e.g. [`sudo add-apt-repository ...`](https://manpages.ubuntu.com/manpages/trusty/man1/add-apt-repository.1.html).

Reply:
would have been better. I think for the next time. Since the VM is already built I would say it is not worth this change.